artifactory/utils/buildutils_test.go

```diff
@@ -0,0 +1,114 @@
+package utils
+
+import (
+	"os"
+	"path/filepath"
+	"strconv"
+	"testing"
+	"time"
+
+	"github.com/jfrog/jfrog-cli-core/v2/utils/coreutils"
+	artclientutils "github.com/jfrog/jfrog-client-go/artifactory/services/utils"
+	"github.com/jfrog/jfrog-client-go/utils/io/fileutils"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+var timestamp = strconv.FormatInt(time.Now().Unix(), 10)
+
+func TestGetBuildName(t *testing.T) {
+	const buildName = "buildName1"
+	const buildNameEnv = "envBuildName"
+	const buildNameFile = "fileBuildName"
+
+	// Setup global build name env var.
+	// Ensure other parallel tests wont get effected.
+	oldBuildName := coreutils.BuildName
+	coreutils.BuildName = oldBuildName + timestamp
+	defer func() { coreutils.BuildName = oldBuildName }()
+
+	// Create build config in temp folder.
+	tmpDir, err := fileutils.CreateTempDir()
+	require.NoError(t, err)
+	defer fileutils.RemoveTempDir(tmpDir)
+
+	// Create build config in temp folder
+	confFileName := filepath.Join(tmpDir, ".jfrog", "projects")
+	assert.NoError(t, fileutils.CopyFile(confFileName, filepath.Join("testdata", "build.yaml")))
+
+	wdCopy, err := os.Getwd()
+	assert.NoError(t, err)
+	assert.NoError(t, os.Chdir(tmpDir))
+	defer os.Chdir(wdCopy)
```

Comment:
Let's not ignore the error returned by ```os.Chdir```. Theer are other places in the code that do not assert the error returned by this function, so let's fix all of them.